### PR TITLE
Version bump

### PR DESCRIPTION
**Pull request by lbargaoanu**
We could do it in the first PR to come, but we'll most likely forget :) Apparently GitHub changed a few things about the issue template. Check https://help.github.com/en/articles/creating-issue-templates-for-your-repository.

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <!-- 
-   If you've just upgraded to 8.0, please read the upgrade guide first (https://docs.automapper.org/en/latest/8.0-Upgrade-Guide.html).
--->
-<!-- For feature requests, just clear out the below 
-Try to provide [a minimal, complete, and verifiable example](https://stackoverflow.com/help/mcve), preferably a [gist](https://gist.github.com/lbargaoanu/9c7233441c3a3413cc2b9b9ebb5964a9) that we can execute and see fail. [Here](https://gist.github.com/lbargaoanu/0cbc531306223f7ffc5468becf2642d6) is an example for ProjectTo.
+
+   If you're new to AutoMapper, please ask a question on StackOverflow first and come back here if the people there consider it a bug.
+   If you've just upgraded to 9.0, please read the upgrade guide first (https://docs.automapper.org/en/latest/9.0-Upgrade-Guide.html).
+   Try to provide [a minimal, complete, and verifiable example](https://stackoverflow.com/help/mcve), preferably a [gist]   (https://gist.github.com/lbargaoanu/9c7233441c3a3413cc2b9b9ebb5964a9) that we can execute and see fail. [Here](https://gist.github.com/lbargaoanu/0cbc531306223f7ffc5468becf2642d6) is an example for ProjectTo.
```
Comment (Tasteful) [inline ISSUE_TEMPLATE.md]: The https://gist.github.com/lbargaoanu/0cbc531306223f7ffc5468becf2642d6 is using the removed `Mapper.Instance`

Comment (lbargaoanu) [reply ISSUE_TEMPLATE.md]: Thanks, I'll change it.

File: docs/Dependency-injection.md [hunk 2]
```diff
@@ -49,7 +49,56 @@ public class EmployeesController {
 	// use _mapper.Map or _mapper.ProjectTo
 }
 ```
+### AutoFac
 
+https://dotnetfalcon.com/autofac-support-for-automapper/
```
Comment (lbargaoanu) [inline docs/Dependency-injection.md]: I don't know anything about AutoFac, but LGTM :) It seems fair to have the link. I didn't ask for permission yet, but I will.

Comment (conwid) [reply docs/Dependency-injection.md]: Thanks for reaching out to me. Go ahead and add it to the official documentation.

Comment (lbargaoanu) [reply docs/Dependency-injection.md]: Thanks!

**Pull request merged**